database/database.json

```diff
@@ -1,10 +1,10 @@
 {
-    "dev": {
-        "driver": "pg",
-        "user": "postgres",
-        "password": "opensesame",
-        "host": "localhost",
-        "database": "cyf_tms",
-        "port": "5432"
-    }
+	"dev": {
+		"driver": "pg",
+		"user": "postgres",
+		"password": "opensesame",
+		"host": "localhost",
+		"database": "cyf_tms",
+		"port": "5432"
+	}
 }
```

Comment:
Seems like you didn't change anything here. Remove this file from commit

Reply:
done